client/my-sites/earn/memberships/products.jsx

```diff
@@ -74,13 +74,18 @@ class MembershipsProductsSection extends Component {
 					{ this.props.translate( 'Payment plans' ) }
 				</HeaderCake>
 
-				{ this.props.hasStripeFeature && (
-					<SectionHeader>
+				<SectionHeader>
+					{ this.props.hasStripeFeature && (
 						<Button primary compact onClick={ () => this.openAddEditDialog( null ) }>
 							{ this.props.translate( 'Add a new payment plan' ) }
 						</Button>
-					</SectionHeader>
-				) }
+					) }
+					{ ! this.props.hasStripeFeature && (
+						<Button primary compact href={ '/plans/' + this.props.siteSlug }>
```

Comment:
Presumably this needs some tracks events adding

Reply:
The stats for plan upgrades on /earn are used on the mission control /s/calypso_earn_page stats, this is powered by bump stats.

There is also the `calypso_earn_page_upgrade_button_click` Tracks stat (properties are plan: wp.com plan or any-paid-plan, feature:donations/recurring-payments/etc).

 /earn/payments-plans is used on donations, premium-content, and manage payments, it doesn't actually know (or care) which of those features was used to get to the page. 

If reusing the `calypso_earn_page_upgrade_button_click` event name, it could use 'any-feature' as the event name analogous to any-paid-plan, though the wording in Tracks all talks about the cards on the /earn page, so perhaps this isn't a good idea. I'm leaning towards coining a new tracks name.

Is the bump stats tracking needed here at all?

Reply:
> I'm leaning towards coining a new tracks name.

I agree.
I'd probably mimic the path with a `calypso_earn_page_payment_plans_upgrade_button_click`, with no properties.

And I'll also keep the bump as well, reusing the original `calypso_earn_page` event, with a `payment-plans-upgrade-button` feature for easy comparison.